app/javascript/app/components/ndcs/ndcs-lts-table/ndcs-lts-table-selectors.js

```diff
@@ -100,30 +98,30 @@ export const tableRemoveIsoFromData = createSelector(
     if (!data || isEmpty(data)) return null;
 
     return data.map(d => {
+      const updatedD = { ...d };
       let date = d['Submission Date'];
       try {
         date = new Date(d['Submission Date']);
         date = !isNaN(date.getTime())
           ? {
-              name: date.toLocaleDateString('en-US'),
-              value: date.getTime()
-            }
+            name: date.toLocaleDateString('en-US'),
+            value: date.getTime()
+          }
           : {
-              name: undefined,
-              value: undefined
-            };
+            name: undefined,
+            value: undefined
+          };
       } catch (e) {
         console.error(e);
       }
-      d['Submission Date'] = date;
-      d['Document'] = d['Document']
-        ? d['Document'].replace('href=', "target='_blank' href=")
+      updatedD['Submission Date'] = date;
```

Comment:
It's more a suggestion but I'd prefer a bit more descriptive names than `updatedD` because it's not that obvious for someone who doesn't know the code. It's still better than the old `d[...]`, but not clear enough IMO.